test/browser_tests/page_objects/page_careers-application-process.js

```diff
@@ -0,0 +1,44 @@
+'use strict';
+
+var _assign = require( 'lodash' ).assign;
```

Comment:
So we we already using lodash so I would recommend using that. I also like the fact that they have broken out the functions at as node [modules](https://www.npmjs.com/package/lodash.assign).